Projects/Embark/Sources/Embark.swift

```diff
@@ -38,11 +38,22 @@ extension Embark: Presentable {
         let scrollView = FormScrollView()
         scrollView.backgroundColor = .brand(.primaryBackground())
         let form = FormView()
-        form.dynamicStyle = DynamicFormStyle.default.restyled { (style: inout FormStyle) in
-            style.insets = .zero
-        }
+
         bag += viewController.install(form, options: [], scrollView: scrollView) { scrollView in
             scrollView.alwaysBounceVertical = false
+
+            bag += scrollView.traitCollectionSignal.onValue { _ in
```

Comment:
should we combine `traitCollection` and `contentSize` here maybe? in case the height changes so much that we scroll, you can test this by changing dynamic type to a really large font-size.